lib/windup.js

```diff
@@ -43,7 +44,7 @@ Windup.prototype.move = function(floors) {
   if (this.status === "falling" && (this.y < this.canvas.height - this.height - this.floorHeight)) {
     this.fall();
   } else if (this.status === "jumping") {
-    new Jump().jump(floors, this);
+    this.jump.jump(floors, this);
  } else if (this.x >= this.dino.x) {
```

Comment:
I think this would be cleaner if we make this.jump refer to a function, instead of a Jump object, so that when we call it using this.jump(args) instead of this.jump.jump(args)